tests/suites/deploy/deploy_charms.sh

```diff
@@ -14,6 +14,23 @@ run_deploy_charm() {
 	destroy_model "test-deploy-charm"
 }
 
+run_deploy_charm_unsupported_series() {
```

Comment:
nice, once this lands I can re-generate the tests for Jenkins and get more green in proving grounds.